rune/libenclave/epm/epm.go

```diff
@@ -86,6 +101,15 @@ func SaveCache(ID string, subtype string, enclaveinfo *v1alpha1.Enclave) error {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
 
+	healthzResp, err := c.Healthz(ctx, &v1alpha1.HealthzRequest{Type: Type})
+	if err != nil {
+		return nil
+	}
+	if healthzResp.Ok != true {
```

Comment:
ditto

Reply:
ok, thanks. Will pr revision this afternoon.